assets/src/scss/pages/post/_comments-form.scss

```diff
@@ -1,12 +1,17 @@
 .comments-block {
+  font-family: var(--font-family-paragraph-secondary);
+
+  .required-field-message {
```

Comment:
This is the default message shown by WordPress for logged in users, to say that fields with an * are required. We no longer need to show it since we use `(required)` instead.